README.md

```diff
@@ -68,6 +66,31 @@ metadata:
 type: Opaque
 ```
 
+The IAM role policy of your key should allow at least the following `operation`s for your domain: `list-dns-domains`, `list-dns-domain-records`, `get-dns-domain-record`, `get-operation`, `create-dns-domain-record` and `delete-dns-domain-record`
```

Comment:
thank you for the changes!

One last request: can you please remove the `get-operation` from the list(again). We have confirmed that it isn't needed anymore. Sorry for the back and forth.

Reply:
Done